docs/apidesc.md

```diff
@@ -66,8 +66,8 @@ one exception is NICs, where the label is unique only on a per-node basis.
     project_connect_node        <project_label> <node_label>
     project_detach_node         <project_label> <node_label>
  
-    node_connect_network        <node_label> <nic_label> <network_label>
-    node_detach_network         <node_label> <nic_label>
+    node_connect_network        <node_label> <nic_label> <network_label> [ <channel> ]
+    node_detach_network         <node_label> <nic_label> <network_label>
```

Comment:
Could we make network_label optional here? In the (common?) case of not using channels, it would just detach the native network and be close to how many users would think of it. It could throw an error if more than one network were connected.